pkgs/applications/audio/dgedit/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchgit
+, pkgconfig
+, autoreconfHook
+, libao
+, libsndfile
+, qtbase
+, qtx11extras
+, qttools
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dgedit";
+  version = "0.10.0";
+
+  src = fetchgit {
+    url = "git://git.drumgizmo.org/dgedit.git/";
+    rev = "v${version}";
+    sha256 = "11mcfa6h61q0gk0ynfpqql4lnih0m3y2z9mdjq4p1219hrapxs5q";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkgconfig autoreconfHook wrapQtAppsHook ];
+  buildInputs = [ qtbase qtx11extras qttools libao libsndfile ];
+
+  meta = with stdenv.lib; {
+    description = "the DrumGizmo drumkit editor";
+    homepage = "https://drumgizmo.org/";
+    license = licenses.gpl3Plus;
```

Comment:
The licensing situation of this project seems a bit mixed. The program (menu Help/about/License) displays a  GPL 3, calls it GPL (without version number), but the source files say "GPL version 2 or later" :confused:. Obviously, GPL 3 is included in "or later". I don't know what we should do here, maybe ask upstream to clarify? Is this actually a problem?

Reply:
Clearification would be nice, yes.